src/heads-up/components/app-tooltip.vue

```diff
@@ -1,152 +1,101 @@
 <template>
-  <span
-    v-tooltip.top="{ ref: 'tooltip' }"
+  <v-popover
+    trigger="hover"
+    :placement="placement"
+    :delay="{ show: 500, hide: 100 }"
```

Comment:
The `hide` in my commits were merely to test the functionality, I think you can leave it out.